universal-application-tool-0.0.1/project/Tailwindbuilder.scala

```diff
@@ -3,20 +3,14 @@ import sbt.File
 
 import scala.sys.process.Process
 
-object Webpack {
+object Tailwindbuilder {
   def apply(base: File): PlayRunHook = {
-    object WebpackHook extends PlayRunHook {
+    object TailwindbuilderHook extends PlayRunHook {
       var process: Option[Process] = None
 
       override def beforeStarted() = {
         process = Option(
-          Process("npx webpack", base).run()
-        )
-      }
-
-      override def afterStarted() = {
-        process = Option(
-          Process("npx webpack --watch", base).run()
+          Process("npx tailwindcss-cli@latest build ./app/assets/stylesheets/styles.css -o ./public/stylesheets/tailwind.css", base).run()
```

Comment:
question (non-blocking): If we ever wanted an additional css file + output file, we'd need to add it to this line and the one below, right?

Reply:
or add it as a completely separate file in app/assets

Reply:
@natsid As much as I understand the Build hook, that is correct. Tailwind CLI being used here to compile that styles.css, which uses @tailwind directives, to tailwind.css. If the CSS files are not utilizing TailwindCLI to compile, there is no need to add it to this line, you can just put inside app/assets as mentioned by Bion or use public directory since any js/css files under public package are automatically packaged and added to class path, typically, asset.jars. Please correct me if I'm wrong.